test/boost/sstable_datafile_test.cc

```diff
@@ -1151,7 +1152,6 @@ SEASTAR_TEST_CASE(sstable_tombstone_metadata_check) {
                     .with_column("ck1", utf8_type, column_kind::clustering_key)
                     .with_column("r1", int32_type)
                     .build();
-            auto tmp = env.tempdir().make_sweeper();
```

Comment:
But why? I placed sweepers in locations where test cases failed stepping on non-empty directory. Why did it become useless?